x-pack/plugins/integration_assistant/server/templates/readme/setup/aws-s3.md.njk

```diff
@@ -0,0 +1,26 @@
+### Collecting logs from Amazon S3 bucket
+
+When collecting logs from S3 bucket is enabled, users can retrieve logs from S3 objects that are pointed to by S3 notification events read from an SQS queue or directly polling list of S3 objects in an S3 bucket. 
+
+The use of SQS notification is preferred: polling list of S3 objects is expensive in terms of performance and costs and should be preferably used only when no SQS notification can be attached to the S3 buckets. This input integration also supports S3 notification from SNS to SQS.
```

Comment:
```suggestion
The use of SQS notification is preferred; polling list of S3 objects is expensive in terms of performance and costs and should be preferably used only when no SQS notification can be attached to the S3 buckets. This input integration also supports S3 notification from SNS to SQS.
```
or
```suggestion
The use of SQS notification is preferred. Polling list of S3 objects is expensive in terms of performance and costs and should be preferably used only when no SQS notification can be attached to the S3 buckets. This input integration also supports S3 notification from SNS to SQS.
```
I prefer the first as it's formally more correct, but semicolons are often avoided, so the latter is a reasonable fallback.